.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "\U00002728 Feature request"
+description: Suggest an idea for this project
+title: ''
+labels: [enhancement]
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Some of these titles are questions (like here), some are imperative (e.g. "Describe the issue") and some are generic titles (e.g. "Expected behaviour"). I think the latter is probably best.